dashboard/app/models/user.rb

```diff
@@ -961,6 +961,7 @@ def password_required?
   # address associated with them because it wasn't required when they were
   # created. Those old accounts are allowed to skip the email validation.
   def teacher_email_required?
+    return false if Policies::Lti.lti? self
```

Comment:
Since teacher's emails are optional on the LMS side, during a user's creation it is now possible for the email to be `nil` but we will prompt the teacher for their email upon login.